ocs_ci/deployment/cnv.py

```diff
@@ -648,3 +648,135 @@ def disable_multicluster_engine(self):
+
+        """
+
+        vm_obj = OCP(kind=constants.VIRTUAL_MACHINE, namespace=namespace)
+        vmi_obj = OCP(kind=constants.VIRTUAL_MACHINE_INSTANCE, namespace=namespace)
+
+        return vm_obj.get(
+            out_yaml_format=False, all_namespaces=not namespace, dont_raise=True
+        ) or vmi_obj.get(
+            out_yaml_format=False, all_namespaces=not namespace, dont_raise=True
+        )
+
+    def remove_hyperconverged(self):
+        """
+        Remove HyperConverged CR
+
+        """
+        hyperconverged_obj = OCP(
+            kind=constants.HYPERCONVERGED,
+            resource_name=constants.KUBEVIRT_HYPERCONVERGED,
+            namespace=self.namespace,
+        )
+        hyperconverged_obj.delete(resource_name=constants.KUBEVIRT_HYPERCONVERGED)
+        logger.info(
+            f"Deleted {constants.HYPERCONVERGED} {constants.KUBEVIRT_HYPERCONVERGED}"
+        )
+
+    def remove_cnv_subscription(self):
+        """
+        Remove CNV subscription
+
+        """
+        cnv_sub = OCP(
+            kind=constants.SUBSCRIPTION,
+            resource_name=constants.KUBEVIRT_HYPERCONVERGED,
+            namespace=self.namespace,
+        )
+        cnv_sub.delete(resource_name=constants.KUBEVIRT_HYPERCONVERGED)
+        logger.info(f"Deleted subscription {constants.KUBEVIRT_HYPERCONVERGED}")
+
+    def remove_cnv_csv(self):
+        """
+        Remove CNV ClusterServiceVersion
+
+        """
+        cnv_csv = OCP(
+            kind=constants.CLUSTER_SERVICE_VERSION,
+            selector=constants.CNV_SELECTOR,
+            namespace=self.namespace,
+        )
+        cnv_csv.delete(resource_name=cnv_csv.get()["items"][0]["metadata"]["name"])
+        logger.info(f"Deleted ClusterServiceVersion {constants.CNV_OPERATORNAME}")
+
+    def remove_cnv_operator(self):
+        """
+        Remove CNV operator
+
+        """
+        cnv_operator = OCP(
+            kind=constants.OPERATOR_KIND, resource_name=constants.CNV_OPERATORNAME
+        )
+        cnv_operator.delete(resource_name=constants.CNV_OPERATORNAME)
+        logger.info(f"Deleted operator {constants.CNV_OPERATORNAME}")
+
+    def remove_crds(self):
+        """
+        Remove openshift virtualization CRDs
+
+        """
+        OCP().exec_oc_cmd(
+            command=f"delete crd -n {self.namespace} -l {constants.CNV_SELECTOR}"
+        )
+        logger.info("Deleted all the openshift virtualization CRDs")
+
+    def remove_namespace(self):
+        """
+        Remove openshift virtualization namespace
+
+        """
+        cnv_namespace = OCP()
+        switch_to_default_rook_cluster_project()
+        cnv_namespace.delete_project(constants.CNV_NAMESPACE)
+        logger.info(f"Deleted the namespace {constants.CNV_NAMESPACE}")
+
+    def uninstall_cnv(self, check_cnv_installed=True):
+        """
+        Uninstall CNV deployment
+
+        Args:
+            check_cnv_installed (Bool): True if want to check if CNV installed
```

Comment:
```suggestion
            check_cnv_installed (bool): True if want to check if CNV installed
```

Reply:
I think the format you are suggesting holds good for return values but not for function arguments. For func arguments we need to mention argument name.

Reply:
Yes. what I mean, it to use arg type as bool instead of Bool. I missed the arg name

Reply:
ack